### PR TITLE
get-wrote-bytes added a simple getter to access the the number of byte…

### PR DESCRIPTION
**Pull request by GregoryCarver**
…s that's been written to file so far.

We appreciate your contributing to Verilator.  If this is your first commit, please add your name to docs/CONTRIBUTORS, and read our contributing guidelines in docs/CONTRIBUTING.rst.

### CODE REVIEW

File: include/verilated_vcd_c.h [hunk 1]
```diff
@@ -126,6 +126,7 @@ class VerilatedVcd VL_NOT_FINAL : public VerilatedTrace<VerilatedVcd, VerilatedV
     // ACCESSORS
     // Set size in megabytes after which new file should be created
     void rolloverMB(uint64_t rolloverMB) { m_rolloverMB = rolloverMB; }
+    uint64_t getWroteBytes() { return m_wroteBytes; }
```
Comment (wsnyder) [inline include/verilated_vcd_c.h:129]:
```suggestion
    uint64_t wroteBytes() { return m_wroteBytes; }
```

Comment (wsnyder) [reply include/verilated_vcd_c.h:129]: Actually should be const.  And if we add this function we'll need a test to (roughly) check it.

Comment (GregoryCarver) [reply include/verilated_vcd_c.h]: Cool, sounds good. I'll try to figure out how to add it and get it done.

**Comment by GregoryCarver** [comment]
I'm going to delete this branch, and go with your suggestion of create a private variable that will determine how many files to create when rolloverMB is created.

**Pull request closed without merging**